apps/web/src/app.css

```
@@ -925,3 +925,63 @@
     transition-duration: 0.01ms !important;
   }
 }
+
+@custom-variant data-open {
+  &:where([data-state="open"]),
```

Stylelint (17.5.0)

[error] Unexpected missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)